Porting/test-dist-modules.pl

```
@@ -7,17 +7,32 @@
 use ExtUtils::Manifest "maniread";
 use Cwd "getcwd";
 
+$|++;
+
 -f "Configure"
   or die "Expected to be run from a perl checkout";
 
+my $github_ci = $ENV{'GITHUB_SHA'} ? 1 : 0;
+
```

Instead of making the continue on failure and github grouping output both depend on `$ENV{'GITHUB_SHA'}` wouldn't it be better to make the first be controlled by a command-line switch?